gitpod-io/gitpod
# [jb]: customize remote indicator

## Description


This PR customizes the remote indicator in the thin client to show Gitpod logo and workspace ID.
<img width="254" alt="Screenshot 2022-02-22 at 06 22 19" src="https://user-images.githubusercontent.com/3082655/155068749-afa61017-af5f-418d-a6e9-4749d6392613.png">

## Related Issue(s)

Fixes https://github.com/gitpod-io/gitpod/issues/8192

## How to test


- Start a workspace: https://ak-jb-remote-indicator.staging.gitpod-dev.com/#referrer:jetbrains-gateway/https://github.com/gitpod-io/spring-petclinic
- Check the remote indicator in the top left corner on the thin client.

## Release Notes

```release-note
NONE
```

## Documentation
